owncloudApp/src/main/res/values/dims.xml

```diff
@@ -62,6 +62,12 @@
     <!-- File List -->
     <dimen name="file_list_progress_bar_height">4dp</dimen>
 
+    <!-- item file grid dimens-->
+    <dimen name="item_file_grid_margin">10dp</dimen>
+    <dimen name="item_file_image_grid_margin">1.5dp</dimen>
```

Comment:
If it is checked, they give almost identical results, not perceptible to the eyes. The problem was that 4 I don't know to what extent it was. Of course it wasn't in dp. So if we put 4dp it would not be correct. Debugging I found that the margin was dp1, but in the print I made it did not come with decimals. I found that when I set 1dp it was a little smaller than the original, almost not noticeable to the eye. Setting 1.5dp I can't find the differences, at least with the naked eye.